sdk/redis/Azure.ResourceManager.Redis/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0 (2024-07-08)
```

Comment:
since we are now at 7/11/2024, could we update the release date?

Reply:
Updated the release date